build/yaml/deployBotResources/js/evaluateDependenciesVariables.yml

```diff
@@ -22,23 +22,29 @@ steps:
         $sourceJSMyGet = "https://botbuilder.myget.org/F/botbuilder-v4-js-daily/npm/"
         $sourceJSv3MyGet = "https://botbuilder.myget.org/F/botbuilder-v3-js-daily/npm/"
         $sourceJSNpm = "https://registry.npmjs.com/"
-        switch -regex ("${{ parameters.registry }}") {
+        $versionNumber = "${{ parameters.version }}"
+        $registry = "${{ parameters.registry }}"
+        $botType =  "${{ parameters.botType }}"
+        switch -regex ($registry) {
           "^($null|MyGet)$" {
-            switch ("${{ parameters.botType }}") {
+            switch ($botType) {
               "SkillV3" { $source = $sourceJSv3MyGet }
               default { $source = $sourceJSMyGet }
             }
           }
-          "Npm" { $source = $sourceJSNpm }
-          default { $source = "${{ parameters.registry }}" }
+          "Npm" { $source = $sourceJSNpm 
+            if ([string]::IsNullOrEmpty($versionNumber)){
+              $versionNumber = "STABLE"
+            }
+          }
+          default { $source = $registry }
         }
-        Write-Host "Source: $source"
```

Comment:
This shouldn't be removed, it shows the source of the registry in the pipeline's output

Reply:
Agree, re added in a new commit